src/Components/UpcomingTable/UpcomingTable.tsx

```diff
@@ -431,14 +467,15 @@ export const UpcomingTable: React.FunctionComponent<UpcomingTableProps> = (
       ref={attributeMenuRef}
       onSelect={(_ev, itemId) => {
         setActiveAttributeMenu(
-          itemId?.toString() as 'Name' | 'Release' | 'Date'
+          itemId?.toString() as 'Name' | 'Type' | 'Release' | 'Date'
         );
         setIsAttributeMenuOpen(!isAttributeMenuOpen);
       }}
     >
       <MenuContent>
         <MenuList>
           <MenuItem itemId="Name">Name</MenuItem>
+          <MenuItem itemId="Type">Type</MenuItem>
           <MenuItem itemId="Release">Release</MenuItem>
           <MenuItem itemId="Date">Date</MenuItem>
```

Comment:
I'd create a constant with these values in an array or array of objects and then create each `MenuItem` via a loop or a separate component.